content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md

```diff
@@ -42,6 +42,24 @@ Occasional messages of this type do not usually indicate a problem. If this type
 - The validator described in the message is behaving maliciously.
 
 
+## async_send failed
+
+The following log message indicates that [StatsD export](configure-statsd.html) failed:
```

Comment:
Suggest "has failed"

Reply:
I disagree given the context here. The phrase "has failed" implies finality. This export process happens periodically as long as the server is running. It's a distinct possibility that the StatsD server or network condition has recovered by the time you see this log message, so exporting has already resumed successfully. I think the simpler "failed" carries that implication better here.